device/pensando/arm64-elba-asic-r0/pmon_daemon_control.json

```diff
@@ -0,0 +1,10 @@
+{
+    "skip_thermalctld": true,
+    "skip_fancontrol": true,
+    "skip_ledd": true,
+    "skip_psud": true,
+    "skip_syseepromd": false,
+    "skip_xcvrd": true,
```

Comment:
i do not knw why we skip xcrvd and pcied

Reply:
This is still work in progress and will be enabled later.